decidim-proposals/app/models/decidim/proposals/proposal.rb

```diff
@@ -93,11 +93,9 @@ def self.user_collection(author)
       # Public: Updates the vote count of this proposal.
       #
       # Returns nothing.
-      # rubocop:disable Rails/SkipsModelValidations
       def update_votes_count
-        update_columns(proposal_votes_count: votes.count)
+        update_columns(proposal_votes_count: votes.count) # rubocop:disable Rails/SkipsModelValidations
```

Comment:
Why this change?

Reply:
My rubocop complains about the code wrapping rubocop comments:

`Lint/UnneededDisable: Unnecessary disabling of` "Whatever Cop"

It's the way I found to make it shut up.

Reply:
Thanks. That was unnecessary.